content/post/columbus-bos.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Beheaded Christopher Columbus Statue In Boston Will Be Removed From North End Park"
+date: 2020-06-10T16:09:33-08:00
+draft: true
+---
+
+The statue of genocidal pillager Christopher Columbus [is to be removed today](https://boston.cbslocal.com/2020/06/10/christopher-columbus-statue-beheaded-boston-massachusetts/amp/?__twitter_impression=true) after being [beheaded overnight](https://www.nbcnews.com/news/us-news/boston-christopher-columbus-statue-beheaded-richmond-statue-thrown-lake-n1229201).
```

Comment:
You can remove everything after `https://boston.cbslocal.com/2020/06/10/christopher-columbus-statue-beheaded-boston-massachusetts/`.

Reply:
```suggestion
The statue of genocidal pillager Christopher Columbus [is to be removed today](https://boston.cbslocal.com/2020/06/10/christopher-columbus-statue-beheaded-boston-massachusetts/).
```

Ok, sure 👍